pyppeteer/connection.py

```diff
@@ -40,14 +68,27 @@ def __init__(
         self._lastId = 0
         self._callbacks: Dict[int, asyncio.Future] = dict()
         self._delay = delay / 1000
+
         self._loop = loop
         self._sessions: Dict[str, CDPSession] = dict()
         self.connection: CDPSession
         self._connected = False
         self._ws = websockets.client.connect(
-            self._url, max_size=None, loop=self._loop, ping_interval=None, ping_timeout=None)
+            self._url,
+            max_size=None,
+            loop=self._loop,
+            ping_interval=None,
+            ping_timeout=None,
+        )
```

Comment:
This becomes irrlevant with the work on launcher.py (where this is used)

Reply:
update to use the new WebsocketTransport class for this.